### PR TITLE
Solidity Developer Survey 2022 Announcement

### PR DESCRIPTION
**Pull request by franzihei**

### CODE REVIEW

File: _posts/2022-12-07-solidity-developer-survey-2022-announcement.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+published: true
+title: 'Solidity Developer Survey 2022 is Live!'
+date: '2022-12-07'
+author: Franziska Heintel
+category: Announcements
+---
+
+It's that time of the year again: Today, we are launching the [Solidity Developer Survey 2022](https://cryptpad.fr/form/#/2/form/view/HuPIRv4gvziSV0dPV1SJncKzYJXTVc8LGCaMfLUoj2c/).
```
Comment (NunoFilipeSantos) [inline _posts/2022-12-07-solidity-developer-survey-2022-announcement.md:10]:
```suggestion
It’s that time of the year. 
Drumroll, please! 🥁🥁🥁 
We are launching the [Solidity Developer Survey 2022](https://cryptpad.fr/form/#/2/form/view/HuPIRv4gvziSV0dPV1SJncKzYJXTVc8LGCaMfLUoj2c/)!
```

Happier tone than "again"

File: _posts/2022-12-07-solidity-developer-survey-2022-announcement.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Before we wrap up 2022 for good, we reach out to collect your feedback and insights!
```
Comment (NunoFilipeSantos) [inline _posts/2022-12-07-solidity-developer-survey-2022-announcement.md:12]:
```suggestion
Before we wrap up 2022 for good, we want to reach out to collect your feedback and insights so we can improve on it!
```

The motivation was missing.

File: _posts/2022-12-07-solidity-developer-survey-2022-announcement.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+**📝 [TAKE THE SURVEY!](https://cryptpad.fr/form/#/2/form/view/HuPIRv4gvziSV0dPV1SJncKzYJXTVc8LGCaMfLUoj2c/) 📝**
+
+You can find the results of last year’s Solidity Developer Survey [here](https://blog.soliditylang.org/2022/02/07/solidity-developer-survey-2021-results/).
```
Comment (NunoFilipeSantos) [inline _posts/2022-12-07-solidity-developer-survey-2022-announcement.md:16]:
```suggestion
You can find the previous results of the Solidity Developer Survey 2021 [here](https://blog.soliditylang.org/2022/02/07/solidity-developer-survey-2021-results/), where 435 people from 73 different countries participated, more than half of the respondents used Solidity for less than a year!
```

Added year explicitly for SEO.
Added hook for people to check out the previous blogpost.

File: _posts/2022-12-07-solidity-developer-survey-2022-announcement.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## About the Survey 🪄
+
+Like in previous years, this survey helps us to further improve the language and compiler, and shape the future roadmap of Solidity. We are curious to hear your feedback on Solidity and learn about your personal coding preferences.
```
Comment (NunoFilipeSantos) [inline _posts/2022-12-07-solidity-developer-survey-2022-announcement.md:20]:
```suggestion
Like in previous years, this survey helps us to improve the language and compiler further and shape the future roadmap of Solidity. Therefore, we want to hear your feedback on Solidity and learn about your coding preferences.
```

File: _posts/2022-12-07-solidity-developer-survey-2022-announcement.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+Last but not least, we moved away from Google forms and are trying an open-source alternative, namely CryptPad, in this iteration. Besides the fact that we prefer opting for open-source, encrypted, self-hostable software over proprietary software, selecting CryptPad specifically was a conscious choice since it allows you to submit your responses fully anonymous (client-side encryption).
+
+## Spread the Word! 📯
+
+This marks the third time we are conducting a structured developer survey and we are hoping to reach even more developers than last year.
+
+**Please help us reach as many Solidity developers as possible by sharing the survey in your networks!**
+
+The results of the survey will be shared on this blog as well as on the [Solidity Twitter](https://twitter.com/solidity_lang) and [Mastodon](https://fosstodon.org/@solidity) accounts in Q1 of next year.
```
Comment (NunoFilipeSantos) [inline _posts/2022-12-07-solidity-developer-survey-2022-announcement.md:47]:
```suggestion
The results of the survey will be shared on this blog as well as on the [Solidity Twitter](https://twitter.com/solidity_lang) and [Mastodon](https://fosstodon.org/@solidity) accounts in Q1 of 2023.
```

**Review by NunoFilipeSantos — CHANGES_REQUESTED** [review]
Thank you for doing the blogspost!
A few fixes suggested

**Comment by franzihei** [comment]
Thanks @NunoFilipeSantos! Will resolve the remaining two things tomorrow morning. :)

**Review by NunoFilipeSantos — APPROVED** [review]
Ship it. 🚢

**Pull request merged**